<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="database" uuid="8baecbf4-41ee-4347-bf0d-94271042dbce">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\silvi\Documents\FIT\pyvenv\database.db</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="_database" uuid="f2b9ccc8-7da9-48f9-a68f-e1ce670ead60">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\silvi\Documents\FIT\pyvenv\_database.db</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="database [2]" uuid="81841890-8859-41d1-b2a7-116fede1fead">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\silvi\Documents\FIT\BP\database.db</jdbc-url>
    </data-source>
  </component>
</project>